auth/config/third_party_config.go

```diff
@@ -10,6 +10,7 @@ type FlyteClientConfig struct {
 	ClientID    string   `json:"clientId" pflag:",public identifier for the app which handles authorization for a Flyte deployment"`
 	RedirectURI string   `json:"redirectUri" pflag:",This is the callback uri registered with the app which handles authorization for a Flyte deployment"`
 	Scopes      []string `json:"scopes" pflag:",Recommended scopes for the client to request."`
+	Audience    string   `json:"audience" pflag:",Audience to use when initiating OAuth2 authorization requests."`
```

Comment:
should this have a default value?

Reply:
This is the dns name of the hosted flyte endpoint. Most of the auth providers like okta provide a way of specifying this during auth server creation and have a default value already available. This property allows to override this default in the auth provider. Since this is dependent on the user environment i haven't added a default value for it and expect to either leave it blank in which case it uses auth server default which is the case for our internal deployments too or have user configured audience value.